src/main/scala/viper/gobra/translator/implementations/components/UnknownValuesImpl.scala

```diff
@@ -0,0 +1,54 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2011-2020 ETH Zurich.
+
+package viper.gobra.translator.implementations.components
+
+import viper.gobra.translator.Names
+import viper.gobra.translator.interfaces.Collector
+import viper.gobra.translator.interfaces.components.UnknownValues
+import viper.silver.{ast => vpr}
+
+class UnknownValuesImpl extends UnknownValues {
```

Comment:
To generate expressions whos value is not known. For example, comparing addresses of zero-sized data may or may not be true. Since we can deduce neither in our verifier, I use this domain.